controller/submit.js

```diff
@@ -91,6 +92,7 @@ function createGist({ project, stats }) {
   content = JSON.stringify(content);
   return new Promise(function (res, rej) {
     console.log(clc.yellow('Creating gist. . .'));
+    //issue with the command, saying that something was not found: Logan V
```

Comment:
```suggestion
```

Reply:
Also not a big deal, but the convention is to put a space between the slashes and the comment text
```javascript
// you can use cmd + / in VS Code to auto-create a comment in most languages
// so you don't even have to think about conventions.
```